OpenKeychain/src/main/res/layout/two_line_spinner_dropdown_item.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingLeft="10dp"
+    android:paddingTop="6dp"
+    android:paddingRight="10dp"
+    android:paddingBottom="6dp"
+    android:background="#FFFFFFFF">
```

Comment:
I simply disliked the gray background which appears. Didn't notice that doing so cancels out the ripple effect. I'll see if I can achieve both.